sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/LoginSteps.java

```diff
@@ -59,8 +59,8 @@ public LoginSteps(
         "I log in with the user",
         () -> {
           webDriverHelpers.accessWebSite(environmentUrl);
-          webDriverHelpers.fillInWebElement(LoginPage.USER_NAME_INPUT, "NatUser");
-          webDriverHelpers.fillInWebElement(LoginPage.USER_PASSWORD_INPUT, "NatUser38118");
+          webDriverHelpers.fillInWebElement(LoginPage.USER_NAME_INPUT, "SurvOff");
+          webDriverHelpers.fillInWebElement(LoginPage.USER_PASSWORD_INPUT, "SurvOff38118");
```

Comment:
changing this user can impact the other tests?

Reply:
Please revert this and use :
```gherkin
I log in as a ([^"]*)
```

Reply:
This is github issue was not changed by me https://github.com/hzi-braunschweig/SORMAS-Project/pull/5476#discussion_r635900392

Reply:
i changed it back